specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/restorePoint.json

```diff
@@ -681,6 +681,10 @@
         "location": {
           "type": "string",
           "description": "Location of the VM from which the restore point was created."
+        },
+        "userData": {
+          "type": "string",
+          "description": "The userData associated with the source VM."
```

Comment:
The description for UserData in the virtualMAchine.json swagger is this:
UserData for the VM, which must be base-64 encoded. Customer should not pass any secrets in here.
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachine.json#L3065

Should this be the same?

Reply:
Thanks @Sandido. I have more descriptive description now based on your suggestion as user Data description should be relative to restore points for this use Case.